extensions/logs/logs.go

```diff
@@ -61,7 +60,7 @@ type Config struct {
 	ArtifactsDir      string        `default:"logs" desc:"Directory for storing container logs" envconfig:"ARTIFACTS_DIR"`
 	Timeout           time.Duration `default:"5s" desc:"Context timeout for kubernetes queries" split_words:"true"`
 	WorkerCount       int           `default:"8" desc:"Number of log collector workers" split_words:"true"`
-	AllowedNamespaces string        `default:"(ns-.*)|(nsm-system)" desc:"Regex of allowed namespaces" split_words:"true"`
+	AllowedNamespaces string        `default:"(^ns-.*)|(nsm-system)|spire|calico-vpp-dataplane" desc:"Regex of allowed namespaces" split_words:"true"`
```

Comment:
Calico is not a default value for the env.

```suggestion
	AllowedNamespaces string        `default:"(^ns-.*)|(nsm-system)|(spire)" desc:"Regex of allowed namespaces" split_words:"true"`
```